.github/workflows/reset-dev.yml

```
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  reset-dev:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)